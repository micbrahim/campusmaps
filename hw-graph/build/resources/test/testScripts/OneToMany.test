# tests a parent node that has edges coming from node to two other nodes.
# tests validity of parent-child relationship

# create empty graph
CreateGraph g1

# add three nodes
AddNode g1 n1
AddNode g1 n2
AddNode g1 n3

# creates two edges coming from the same node to two different nodes
AddEdge g1 n1 n2 e1
AddEdge g1 n1 n3 e2

# list children of parent node
ListChildren g1 n1