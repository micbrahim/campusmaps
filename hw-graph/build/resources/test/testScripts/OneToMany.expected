# tests a parent node that has edges coming from node to two other nodes.
# tests validity of parent-child relationship

# create empty graph
created graph g1

# add three nodes
added node n1 to g1
added node n2 to g1
added node n3 to g1

# creates two edges coming from the same node to two different nodes
added edge e1 from n1 to n2 in g1
added edge e2 from n1 to n3 in g1

# list children of parent node
the children of n1 in g1 are: n2(e1) n3(e2)